src/lib/components/overlayMenu/elements/Button.svelte

```diff
@@ -6,17 +6,23 @@
 	export let icon: ComponentType;
 
 	export let text: string;
+
+	export let backgroundColor: string = "white";
```

Comment:
might clash depending on css theme?

Reply:
The component was built to be white by default before, so when changing it, it was set to white by default again to not break the sidepanels and topbar